common/buildcraft/lib/gui/GuiBC8.java

```diff
@@ -73,11 +73,11 @@ protected final void drawGuiContainerBackgroundLayer(float partialTicks, int mou
 
     @Override
     protected final void drawGuiContainerForegroundLayer(int mouseX, int mouseY) {
+        drawForegroundLayer();
+
         GlStateManager.translate(-guiLeft, -guiTop, 0);
         mouse.setMousePosition(mouseX, mouseY);
 
-        drawForegroundLayer();
-
```

Comment:
Sorry, it was changed to align drawForegroundLayer with drawBackgroundLayer so they would both start from the same co-ordinates. So can you change it back?